lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -645,22 +644,39 @@ class ConvertTritonGPUOpToLLVMPattern
     return multiDimIdx;
   }
 
+
+  struct SmallVectorKeyInfo {
+    static unsigned getHashValue(const SmallVector<unsigned> &key) {
+      return llvm::hash_combine_range(key.begin(), key.end());
+    }
+    static bool isEqual(const SmallVector<unsigned> &lhs,
+                        const SmallVector<unsigned> &rhs) {
+      return lhs == rhs;
+    }
+    static SmallVector<unsigned> getEmptyKey() {
+      return SmallVector<unsigned>();
+    }
+    static SmallVector<unsigned> getTombstoneKey() {
+      return {std::numeric_limits<unsigned>::max()};
+    }
+  };
+
   SmallVector<SmallVector<Value>>
   emitIndicesForSliceLayout(Location loc, ConversionPatternRewriter &rewriter,
                             const SliceEncodingAttr &sliceLayout,
                             ArrayRef<int64_t> shape) const {
     auto parent = sliceLayout.getParent();
     unsigned dim = sliceLayout.getDim();
     size_t rank = shape.size();
-    auto paddedIndices =
+    auto parentIndices =
         emitIndices(loc, rewriter, parent, sliceLayout.paddedShape(shape));
-    unsigned numIndices = paddedIndices.size();
-    SmallVector<SmallVector<Value>> resultIndices(numIndices);
-    for (unsigned i = 0; i < numIndices; ++i)
-      for (unsigned d = 0; d < rank + 1; ++d)
-        if (d != dim)
-          resultIndices[i].push_back(paddedIndices[i][d]);
-
+    unsigned numIndices = parentIndices.size();
+    SmallVector<SmallVector<Value>> resultIndices;
+    for (unsigned i = 0; i < numIndices; ++i){
```

Comment:
Seems need a clang-format, code style is not right.

Reply:
It seems to pass clang-format on both the CI and locally. Do you have specific formatting options in mind?

Reply:
That's weird, a space seems missing between `++i){`, normally it should be `++i) {` like the following line
https://github.com/openai/triton/blob/triton-mlir/lib/Conversion/TritonGPUToLLVM/TritonGPUToLLVM.cpp#L154

Anyway, I think it is not a major issue, the clang-format is not consistent between versions.